javascript-loops/main.js

```diff
@@ -0,0 +1,62 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+
+function getNumbersToTen() {
+  let currentNumber = 1;
+  const numbers = [];
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  } return numbers;
+}
+
+function getEvenNumbersToTwenty() {
+  const evenNumbers = [];
+  let currentNumber = 2;
+  while (currentNumber <= 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  } return evenNumbers;
+}
+
+function repeatWord(word, times) {
+  let count = 1;
+  let repeated = '';
+  while (count < times) {
+    repeated += word;
+    count++;
+  } return repeated;
+}
```

Comment:
This is returning one less times than it should

Reply:
So for line 31 it would be `(count <= times)`